docs/workflows/android-specific-workflow-steps/android-build.md

```diff
@@ -71,3 +71,39 @@ So, in order to sign your app using the keystore selected in the build configura
 To access the source code of this component, please use the following link:
 
 https://github.com/appcircleio/appcircle-android-build-component.git
+
+## FAQ
+
+### How can I solve the `Out of memory error: Java heap memory` or set the heap memory during the build?
+
+To address this issue, you can use one of the following methods:
+
+#### Method 1: Using a Custom Script
+
+If your project has varying heap size requirements, you can adjust the heap size during the build phase with a [**Custom Script**](/workflows/common-workflow-steps/custom-script) before the **Android Build** step. Your script should include the following command:
+
+```bash
+echo "org.gradle.jvmargs=-Xms1g -Xmx7168M"  >> $AC_REPOSITORY_DIR/gradle.properties
+```
+
+- Adjust the `-Xms1g -Xmx7168M` values according to your project needs.
```

Comment:
@ftmzhrsckldr I could not understand properly what is the meaning of parameters. Could you please give an sentence as example. For ex; `-Xms1g` means bla bla bla, and the other parameter means bla bla bla